examples/prisma/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "name": "tinyhttp-prisma",
+  "type": "module",
+  "scripts": {
+    "start": "node -r esm -r ts-node/register ./src/index.ts"
+  },
+  "dependencies": {
+    "@tinyhttp/app": "workspace:*",
```

Comment:
Should we use a published version instead? This makes sure that people that use `npm`or `yarn` can also run the example. E.g. the `vercel` example points to a released version.

Reply:
we shouldn't because it's a pnpm monorepo

when someone wants to setup an example he/she just needs to remove the `workspace` thing

In the future there will be a CLI to setup an example with a npm version by default

Reply:
@talentlessguy ok then the PR is ready for your review.